Standalone/PythonTests/Automated/benchmark_data_aggregator.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from argparse import ArgumentParser
+import json
+from pathlib import Path
+import time
+import copy
+import subprocess
+import requests
+
+from ly_test_tools.mars.filebeat_client import FilebeatClient
+
+class BenchmarkDataAggregator:
+    def __init__(self, workspace, logger, test_suite):
+        self.build_dir = Path(workspace.paths.build_directory())
+        self.results_dir = Path(workspace.paths.project()) / 'user' / 'Scripts' / 'PerformanceBenchmarks'
```

Comment:
here and throughout is using `Path()` necessary? wouldn't this work just the same (and match our existing patterns for paths):
`self.results_dir = os.path.join(workspace.paths.project(), 'user', 'Scripts', 'PerformanceBenchmarks')`